SMA_FluxExtraction.ipynb

```diff
@@ -0,0 +1,634 @@
+{
```

Comment:
<p>When building getbeam(), it could be useful to explain what the returned information is and how it will later be used. This is so that a user can better manipulate this data for other models in the future.</p><p> </p><p>Karto explained that when an image is decomposed from the synthesized beam, the cleaned map is convolved with a fitted Gaussian model, which is defined by these parameters.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>